### ISSUE TITLE
Finalize Testing Plan for EOY fundraising

### ISSUE THREAD

**Issue by valianttry** [MozillaFoundation/Advocacy#619]
Detailed description of work:
I just walked through lots of testing tickets! They are all great, but we can't do them all in the time we have. 

We need a testing plan for the remainder of the year to optimize for our campaign KPIs. If you need help making decisions @vojtechmzl ping me!

*Start date: November 1, 2016* 
*Due date: November 10, 2016*

**Comment by vojtechsedlak** [comment]
Hi @valianttry, thanks for flagging this. Steph, Scott and I met a while back to chat through the tests and we narrowed things down into the [following list](https://docs.google.com/spreadsheets/d/1Ihcus3J_25xLzw-bQeEBXuIY5X2Q3FD1MG0T1eRaQx4/edit#gid=0), which I ranked based on the methodology used by Adam last year. I will work on fleshing out a timeline and required effort for implementing these tests, so that we can make any necessary adjustments.
